src/lib/optimizer/join_ordering/join_graph_builder.hpp

```diff
@@ -77,7 +77,7 @@ class JoinGraphBuilder final {
    * Returns whether a node of the given type is a JoinGraph vertex in all cases. This is true for all node types that
    * aren't Predicates, Joins or Unions.
    */
-  bool _lqp_node_type_is_vertex(const LQPNodeType node_type) const;
+  static bool _lqp_node_type_is_vertex(const LQPNodeType node_type);
```

Comment:
Not specific to this change: why do we transform all these static methods to private static ones?

Reply:
Because clang-tidy and [Scott Meyers](https://stackoverflow.com/a/11852362/2204581) want us to. ;)

Reply:
Dictators...